MapboxNavigation/NavigationViewController+Obsolete.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+import MapboxCoreNavigation
+import MapboxDirections
+
+//MARK: - Obsoleted Interfaces
+
+public extension NavigationViewController {
+    @available(*, deprecated: 0.1, renamed: "init(route:options:)", message: "Use the new NavigationOptions initalizer.")
```

Comment:
Does the `renamed:` annotation result in a fix-it suggestion? Does it work correctly, considering that the new initializer takes a different set of arguments?

Reply:
Doh, I think it does. Removed.